core/src/test/scala/unit/kafka/server/RequestQuotaTest.scala

```diff
@@ -433,7 +434,7 @@ class RequestQuotaTest extends BaseRequestTest {
       case ApiKeys.FIND_COORDINATOR => new FindCoordinatorResponse(response).throttleTimeMs
       case ApiKeys.JOIN_GROUP => new JoinGroupResponse(response).throttleTimeMs
       case ApiKeys.HEARTBEAT => new HeartbeatResponse(response).throttleTimeMs
-      case ApiKeys.LEAVE_GROUP => new LeaveGroupResponse(response).throttleTimeMs
+      case ApiKeys.LEAVE_GROUP => new LeaveGroupResponse(response, 2).throttleTimeMs
```

Comment:
Does it make sense to use the latest version of LeaveGroupResponse here?  Something like `LeaveGroupResponseData.SCHEMAS.length - 1`

Reply:
Sounds good to me!

Reply:
I guess I'm OK with this as-is for now.